src/modules/navigator/navigator_main.cpp

```diff
@@ -774,7 +769,8 @@ Navigator::run()
 	}
 }
 
-void Navigator::geofence_breach_check(bool &have_geofence_position_data)
+void
```

Comment:
Not that we have a style guide, but I've been in the habit of keeping things on the same line as suggested by the google c++ style guide. Do you have a preference?
https://google.github.io/styleguide/cppguide.html#Function_Declarations_and_Definitions

Reply:
I definitely would like to stick to that style.  I'll push up the inverse of the changes.  I only standardized against the dominant style in the file initially to make a smaller diff.

Reply:
Done! Thank you for the preference suggestion, it feels cleaner!